notes.md

```diff
@@ -0,0 +1,4 @@
+curl -L -o /usr/local/bin/amm-2.11 https://git.io/v7cxd && sudo chmod +x /usr/local/bin/amm-2.11
```

Comment:
What is this file?

Reply:
Ammonite executable; we download and install ammonite. But, specifically, ammonite for Scala 2.11